docs/content/development/extensions-core/approximate-histograms.md

```diff
@@ -1,174 +0,0 @@
-<!--
```

Comment:
why this file is renamed ? we can not see any diff of what this PR is adding to the doc....

Reply:
it's not just approximate histograms now, hence the name change, only the section on fixed histograms is new

Reply:
in the end of the day every thing is an approximate as far i can tell, please revert the change or add a new file for the new work, i want to be able to read what is new in this approach how it is different from others, when can be used, especially that now we have handful of histograms.

Reply:
Sure, I've reverted the file rename.